### PR TITLE
Upgrade to Ember 3.1.0

### PR DESCRIPTION
**Pull request by dmuneras**
## Description

Hey @miguelcobain,

This PR just upgrade the ember version to 3.1.0.

### Motivation

I was having some problems to use your addon in my application. It was saying something like: `Couldn't find module @ember/object imported from blablabla...`.

These are the ember and ember data versions of my application:

```
Ember Inspector | 3.1.0
-- | --
Ember | 2.18.0
Ember Data | 2.18.1
jQuery | 3.3.1


``` 
I was able to fix the problem by updating the ember version of the addon. It looks like the problem is related with the `ember-source` or `ember-cli-babel`.

After doing it everything is working perfectly in my application.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,6 +51,7 @@
   },
   "ember-addon": {
     "configPath": "tests/dummy/config",
-    "demoURL": "https://miguelcobain.github.io/ember-fullscreen/"
+    "demoURL": "https://miguelcobain.github.io/ember-fullscreen/",
+    "after": "ember-cli-shims"
```
Comment (miguelcobain) [inline package.json]: Why is this line needed?

Comment (dmuneras) [reply package.json]: It is not needed. I added that line in a previous commit to make a test, but I removed it.

**Comment by dmuneras** [comment]
Hey @miguelcobain,

I removed the commit which added the change you asked about to avoid confusions.

Best regards,
Daniel.

**Comment by miguelcobain** [comment]
You're totally right, @dmuneras. Didn't see that commit was reverted. My bad.

This looks good to me. 👍 
Thanks!

**Comment by miguelcobain** [comment]
Published 1.0.3 with this. Let me know if something's wrong.

**Comment by dmuneras** [comment]
@miguelcobain  just update my `package.json` and everything works. Thanks a lot.
